core/src/main/resources/org/apache/spark/ui/static/historypage-template.html

```diff
@@ -77,12 +77,12 @@
   <tbody>
   {{#applications}}
     <tr>
+      {{#attempts}}
       <td {{#hasMultipleAttempts}}style="background-color:#fff"{{/hasMultipleAttempts}}>{{version}}</td>
-      <td {{#hasMultipleAttempts}}style="background-color:#fff"{{/hasMultipleAttempts}}><span title="{{id}}"><a href="{{uiroot}}/history/{{id}}/{{num}}/jobs/">{{id}}</a></span></td>
+      <td {{#hasMultipleAttempts}}style="background-color:#fff"{{/hasMultipleAttempts}}><span title="{{id}}"><a href="{{uiroot}}/history/{{id}}{{#hasAttemptId}}/{{attemptId}}{{/hasAttemptId}}/jobs/">{{id}}</a></span></td>
```

Comment:
For my info, what do the # and / prefixes mean here?

Reply:
`{{#...}}` means section begins, `{{/...}}` means section ends. for example mustache tags `{{#hasAttemptId}}/{{attemptId}}{{/hasAttemptId}}`, if `hasAttemptId` is `true`, `/{{attemptId}}` will be rendered.